tests/app/views/contexts/test_submitted_response_context.py

```diff
@@ -65,17 +69,29 @@ def test_build_view_submitted_response_context_submitted_text(app: Flask):
 
 def test_build_view_submitted_response_context_submitted_text_social(app: Flask):
     with app.app_context():
-        questionnaire_store = fake_questionnaire_store()
+        questionnaire_store = fake_questionnaire_store({"tx_id": "tx_id"}, SUBMITTED_AT)
         context = build_view_submitted_response_context(
             "en", SCHEMA, questionnaire_store, SurveyType.SOCIAL
         )
 
         assert context["submitted_text"] == "Answers submitted."
 
 
+def test_build_view_submitted_response_context_submitted_text_health(app: Flask):
```

Comment:
As above, can just be parametrised since only the theme is different. Both schema should enable view submitted response for consistency.

Reply:
Added, thanks.